assemblies/assembly_dmn-models/main.adoc

```diff
@@ -12,7 +12,7 @@ include::_artifacts/author-group.adoc[]
 // Purpose statement for the assembly
 As a business analyst or business rules developer, you can use Decision Model and Notation (DMN) to model a decision service graphically in a decision requirements diagram (DRD). This diagram consists of one or more decision requirements graphs (DRGs) that trace business decisions from start to finish, with each decision node using logic defined in DMN boxed expressions such as decision tables.
 
-{PRODUCT} provides design and runtime support for DMN 1.2 models at conformance level 3, and runtime-only support for DMN 1.1 models at conformance level 3. You can design your DMN models directly in {CENTRAL} or import existing DMN models into your {PRODUCT} projects for deployment and execution. Any DMN 1.1 models that you import into {CENTRAL}, open in the DMN designer, and save are converted to DMN 1.2 models.
+{PRODUCT} provides design and runtime support for DMN 1.2 models at conformance level 3, and runtime-only support for DMN 1.1 and 1.3 models at conformance level 3. You can design your DMN models directly in {CENTRAL} or import existing DMN models into your {PRODUCT} projects for deployment and execution. Any DMN 1.1 models that you import into {CENTRAL}, open in the DMN designer, and save are converted to DMN 1.2 models. DMN 1.3 in DMN designer is not supported in {CENTRAL}.
```

Comment:
Suggest "... models. DMN 1.3 models are not supported in the DMN designer in {CENTRAL}." All cases where applicable.